conjure-java-core/src/integrationInput/java/com/palantir/product/NameCollisionServiceEndpoints.java

```diff
@@ -63,23 +63,22 @@ private static final class IntEndpoint implements HttpHandler, Endpoint {
         public void handleRequest(HttpServerExchange exchange) throws IOException {
             AuthHeader authHeader = runtime.auth().header(exchange);
             String deserializer_ = deserializer.deserialize(exchange);
-            runtime.markers().param("com.palantir.logsafe.Safe", "deserializer", deserializer_, exchange);
             Map<String, String> pathParams =
```

Comment:
This line was removed because it's a request body, which are no longer included.